### PR TITLE
Removing crypto

### PR DESCRIPTION
**Pull request by jteheran**

### CODE REVIEW

**Comment by jteheran** [comment]
@DrDaveD please advise on the name of the new files: cert_utils.py and cert_request.py

File: osgpkitools/incommon_request.py [hunk 1]
```diff
@@ -346,18 +349,19 @@ def main():
         CONFIG = dict(config_parser.items('InCommon'))
 
         ARGS = parse_args()
+
         utils.check_permissions(ARGS['certdir'])
         
         # Creating SSLContext with cert and key provided
         # usercert and userprivkey are already validated by utils.findusercred
-        ssl_context = utils.get_ssl_context(usercert=ARGS['usercert'], userkey=ARGS['userprivkey'])
+        ssl_context = cert-utils.get_ssl_context(usercert=ARGS['usercert'], userkey=ARGS['userprivkey'])
```
Comment (DrDaveD) [inline osgpkitools/incommon_request.py]: This should be cert_utils, right?

File: osgpkitools/incommon_request.py [hunk 2]
```diff
@@ -380,7 +384,7 @@ def main():
             sans = host[1:]
             
             utils.charlimit_textwrap('CN: %s, SANS: %s' % (common_name, sans))
-            csr_obj = utils.Csr(common_name, ARGS['certdir'], altnames=sans)
+            csr_obj = cert-utils.Csr(common_name, ARGS['certdir'], altnames=sans)
```
Comment (DrDaveD) [inline osgpkitools/incommon_request.py]: Likewise here.

**Comment by DrDaveD** [comment]
Those new filenames are fine.

I note that this includes the changes from #48 so that one has to be merged first.

If that branch changes as a result of comments before merge, rebase this branch on that branch again, and do git push -f.

**Comment by jteheran** [comment]
@DrDaveD please review new commit. All common code lives now in cert_utils.py.

**Comment by jteheran** [comment]
I just removed it.

**Pull request merged**